securedrop/tests/conftest.py

```diff
@@ -121,26 +125,25 @@ def setup_gpg(gpg_dir: Path) -> str:
     journalist_public_key_path = Path(__file__).parent / "files" / "test_journalist_key.pub"
     journalist_public_key = journalist_public_key_path.read_text()
     journalist_key_fingerprint = gpg.import_keys(journalist_public_key).fingerprints[0]
-
-    # TODO(AD): Don't import the journalist secret key; will be removed in my next PR
-    journalist_secret_key_path = Path(__file__).parent / "files" / "test_journalist_key.sec"
-    journalist_secret_key = journalist_secret_key_path.read_text()
-    gpg.import_keys(journalist_secret_key)
-
```

Comment:
Important change here: the journalist's private/secret key is no longer imported by default.

Having the journalist secret key available in GPG was making the test suite behave differently compared to web app. For example, data in tests could get decrypted using the journalist secret key instead of the expected source key.